src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -16564,5 +16565,836 @@ public static void Main()
             var x2 = nodes.OfType<VariableDeclaratorSyntax>().Skip(1).First();
             Assert.Equal("<anonymous type: (System.Int32 c, System.Int32) Tuple> x2", model.GetDeclaredSymbol(x2).ToTestDisplayString());
         }
+
+        [Fact]
+        public void DefiniteAssignment001()
+        {
+            var source = @"
+using System;
+class C
+{
+        static void Main(string[] args)
+        {
+            (string A, string B) ss;
+
+            ss.A = ""q"";
+            ss.Item2 = ""w"";
+
+            System.Console.WriteLine(ss);
```

Comment:
the line splitting is not a problem with either Write or WriteLine since test harness trims leading/trailing whitespaces.
I've changed the expectedOutput to be one line.